helpdesk_mgmt_ticket_start_date/hooks.py

```diff
@@ -0,0 +1,12 @@
+# © 2020 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See https://www.gnu.org/licenses/lgpl-3.0.html
+from odoo import api, SUPERUSER_ID
+
+
+def post_init_hook(cr, registry):
+    env = api.Environment(cr, SUPERUSER_ID, {})
+    HelpdeskTicket = env['helpdesk.ticket']
+    rec = HelpdeskTicket.search([('start_date', '=', False)])
+
+    for req in rec:
+        rec.start_date = rec.create_date
```

Comment:
When you install, an error is thrown (an "expected singleton" error).
This worked for me (variables were wrong assigned):
```suggestion
    tickets = HelpdeskTicket.search([('start_date', '=', False)])

    for rec in tickets:
        rec.start_date = rec.create_date
```